### DISCUSSION TITLE
HERO 11 - Reccomendation to change code sequence for stabilization

### DISCUSSION THREAD

**Discussion by LADIDO** [gopro/labs#262] [General]
My experience is to change the sequence in the command for QR. 
I prefer first to place the stabilization EIS mode and after the lens - FOV. 

When the sequence is as in the qr code generator, (first LENS and than EIS), the gopro set the LENS mode randomly .
Be aware not all EIS stabilization modes are allowed for choosen FOV.
After I've changed the sequence of the command, gopro setting works perfect.

This was observed when changing stabilization modes 
e0 - off FOV - linear
e1 - on FOV - linear
e2 - high FOV linear+horizont leveling

After switching the configurations sometimes for e0 and e1 gopro was set WIDE instead od LINEAR.

**Comment by dnewman-gpsw** [comment]
Which camera model, which firmware?  The behavior is camera dependent, and I expect not random.  Some lens modes do not have all the stabilization options. e.g. horizon leveling is a form of stabilization, so fHe0 is effectively meaningless, (lens Horizon level, EIS off.)  If the commands work through menu operations on camera, they work as QR Codes.
